package.json

```diff
@@ -60,6 +58,8 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "codemirror": "^5.39.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
```

Comment:
These should be in `devDependencies`. They are only used for tests.

Reply:
I realized it right after git push. Just a sec